testsupport/wait/member.go

```diff
@@ -346,20 +346,13 @@ func UntilSpaceRequestHasNamespaceAccess(subSpace *toolchainv1alpha1.Space) Spac
 				return false
 			}
 
-			for _, nsAccess := range actual.Status.NamespaceAccess {
-				// check the name of the namespaces are matching
-				for _, expectedNamespace := range expectedNames {
-					found := false
-					if expectedNamespace == nsAccess.Name {
-						found = true
-						break
-					}
-					if !found {
-						return false
-					}
-				}
+			// check if the name of namespaces matches
+			var actualNamespaces []string
+			for _, actualNamespace := range actual.Status.NamespaceAccess {
+				actualNamespaces = append(actualNamespaces, actualNamespace.Name)
 			}
-			return true
+
+			return reflect.DeepEqual(expectedNames, actualNamespaces)
```

Comment:
Is order important when using DeepEqual? If yes then you'll need to sort both expectedNames and actualNames before comparing them.

Reply:
I think yes. Thank you so much for noticing it! I will address the change!